plugin/ale/sign.vim

```diff
@@ -120,6 +126,13 @@ function! ale#sign#SetSigns(buffer, loclist)
         exec 'sign unplace ' . current_id . ' buffer=' . a:buffer
     endfor
 
+	" set colot of SignColumn
+	if len(signlist) > 0
+		execute ':highlight SignColumn ctermbg=' . g:ale_sign_term_color . ' guibg=' .g:ale_sign_gui_color
+	else
+		execute ':highlight SignColumn ctermbg=' . g:ale_sign_gui_color_no_error . ' guibg=' . g:ale_sign_gui_color_no_error
+	endif
```

Comment:
The indentation looks a bit screwy here. You should `:set expandtab` for editing these files.